snorkel/contrib/fonduer/fonduer/parser.py

```diff
@@ -162,7 +165,7 @@ def apply(self, x, **kwargs):
                 pass
             # Add visual attributes
             filename = self.pdf_path + document.name
-            create_pdf = not os.path.isfile(filename + '.pdf') and not os.path.isfile(filename + '.PDF')
+            create_pdf = not os.path.isfile(filename + '.pdf') and not os.path.isfile(filename + '.PDF') and not os.path.isfile(filename)
```

Comment:
Do we have PDF files with no extension? Seems strange to assume something is a PDF without an extension.